tools/wasptool

```diff
@@ -368,6 +368,13 @@ def handle_upload(c, fname, tname):
 
         c.expect('>>> ')
 
+def send_notification(c, title, body):
+    msg= {'title': title, 'body': body}
+    c.sendline(f'wasp.system.notifications.__setitem__(1,  {msg})')
```

Comment:
I think this will replace the notification instead of adding one. Maybe you could generate a random id or have the user specify it.

Also you should probably use `wasp.system.notify(id, {msg})`